src/EditorFeatures/VisualBasicTest/InvertLogical/InvertLogicalTests.vb

```diff
@@ -0,0 +1,122 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.CodeRefactorings
+Imports Microsoft.CodeAnalysis.VisualBasic.InvertLogical
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.InvertLogical
+    Partial Public Class InvertLogicalTests
```

Comment:
it will not trigger (currently by design).  i think the non-shortcircuiting operators are rare enough to not care about for now :)

Reply:
Ok. That's fine.
Another test idea: test with selected span (looks like it will work if the proper range is selected)

---
In reply to: [218196725](https://github.com/dotnet/roslyn/pull/29928#discussion_r218196725) [](ancestors = 218196725)

Reply:
done.